pkg/controllers/configobservation/routersecret/observe_router_secret.go

```diff
@@ -53,16 +69,15 @@ func ObserveRouterSecret(genericlisters configobserver.Listers, recorder events.
 	return observedConfig, errs
 }
 
-func routerSecretToSNI(routerSecret *corev1.Secret) ([]interface{}, error) {
+func routerSecretToSNI(routerSecret *corev1.Secret, certFile string, keyFile string) ([]interface{}, error) {
 	certs := []interface{}{}
 	// make sure the output slice of named certs is sorted by domain so that the generated config is deterministic
 	for _, domain := range sets.StringKeySet(routerSecret.Data).List() {
 		certs = append(certs, map[string]interface{}{
 			"names":    []interface{}{"*." + domain}, // ingress domain is always a wildcard
-			"certFile": interface{}("/var/config/system/secrets/v4-0-config-system-router-certs/" + domain),
-			"keyFile":  interface{}("/var/config/system/secrets/v4-0-config-system-router-certs/" + domain),
+			"certFile": interface{}(certFile + domain),
+			"keyFile":  interface{}(keyFile + domain),
```

Comment:
the docs to `ServingCertKeyPairSecret` say `// ServingCertKeyPairSecret is a reference to a secret in namespace/openshift-config that is a kubernetes tls secret.` so this is most probably not what the cert/key files are going to look like. Instead, they are likely to be `tls.crt` and `tls.key`